CONTRIBUTING.md

```
@@ -0,0 +1,149 @@
+# Contributing to Papermark
+
+First off, thank you for considering contributing to Papermark! It's people like you that make Papermark such a great open-source project. This document provides a detailed guide on how to set up the project locally for development, branch naming conventions, PR guidelines, and more.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Local Development Setup](#local-development-setup)
+  - [Prerequisites](#prerequisites)
+  - [Environment Variables](#environment-variables)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)